microsoft/testsuites/acc/cvm.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.environment import EnvironmentStatus
+from lisa.tools import Dmesg, Lsvmbus
+
+
+@TestSuiteMetadata(
+    area="ACC_CVM",
+    category="functional",
+    description="""
+    This test suite ensures configuration and devices for CVM
+    """,
+)
+class CVMSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case verifies that lsvmbus only shows devices
+        that are allowed in a CVM guest
+
+        Steps:
+        1. Call lsvmbus
+        2. Iterate through list returned by lsvmbus to ensure all devices
+           listed are included in valid_class_ids
+        """,
+        priority=1,
```

Comment:
Should it be P1 test cases? What's the impact if the test case is failed? Please work with @LiliDeng to evaluate right priority.

Reply:
this case also applies to non-cvm, @Jose-Teuttli by any chance, you see any issues related with this case?

Reply:
I don't follow how this applies to non-cvm. All guests get offered devices, but CVM guest should reject devices not allowed in isolation. From what I understand, it is on the guest to reject these devices because the host will offer both valid and invalid devices. Which is seen in a line(s) such as the following:
[    0.563976] ubuntu kernel: hv_vmbus: Invalid offer 2 from the host supporting isolation
The devices allowed in isolation is taken from .allowed_in_isolation=true in https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/hv/channel_mgmt.c

With regards to the priority, I am not confident. If the config test fails, then the guest is not properly configured as a CVM. If the devices test fails, then the guest has accepted an invalid device. However, I don't think there is anything stopping the CVM from working if it accepts an invalid device or has improper configuration, if it first passed the smoke test. However, if this were used in validating CVM SKU then it should be investigated.